node-repository/src/main/java/com/yahoo/vespa/hosted/provision/Node.java

```diff
@@ -262,6 +270,8 @@ private Allocation requireAllocation(String message) {
     /** Returns all the reports on this node. */
     public Reports reports() { return reports; }
 
+    public Optional<String> modelId() { return modelId; }
```

Comment:
Please add javadoc explaining what this is as the field name is not very descriptive on its own.

Reply:
Should probably expose this field in `NodesResponse` as well?